src/txn/txn_ckpt.c

```diff
@@ -2541,6 +2541,9 @@ static void
 __checkpoint_timing_stress(WT_SESSION_IMPL *session, uint64_t flag, struct timespec *tsp)
```

Comment:
Out of curiosity, why does checkpoint have a specific timing stress function instead of reusing `__wt_timing_stress`? Is it only because we want to specify the time for `__wt_sleep`?

Reply:
The `__wt_timing_stress` function used to take the time structure and had code to sleep but it was unused in that *every* caller passed in NULL. That is why I removed it. AFAICT the checkpoint stress function exists for checking the checkpoint sessions in the connection.